### ISSUE TITLE
calculate % of features unique to a sample

### ISSUE THREAD

**Issue by rmflight** [rmflight/visualizationQualityControl#9]
It seems that another indicator of problems would be the percentage of all of the features that are sample specific.

For example, if a sample has a large number of features that are *only* in that sample and no other, then we expect there could be a problem.

**Comment by rmflight** [comment]
@jesudk2, it looks like we had this issue with a sample in one of our analyses that we could test this on, correct?

**Comment by jesudk2** [comment]
yes there is one sample that has features that are sample specific.  I can check against that sample and see what percentage there is.

**Comment by rmflight** [comment]
The idea is that I'll write a function in this pkg to do the calculations. I just wanted to confirm that we had an example in a data set to test this with. If you want to do an initial calculation to see if this metric might even be useful, that would be good.

**Comment by jesudk2** [comment]
I will work on that today.  I think it will be extremely useful though
considering the blinded validation studies being performed.  The more
information available for QA/QC and potential confounding issues of samples
will be good to have don't you think?

**Comment by rmflight** [comment]
Yes, I think so. I'm also interested to see how this sample truly compares to the others, and how useful this metric will actually be. We have an indication of it given how many features disappear when we remove that sample from consideration at the beginning, but this calculation actually gives us a firm number / fraction, and we can see if it really is an outlier with respect to this value.
